classes/Setup/class.ilOpenCastDBUpdateSteps.php

```diff
@@ -82,4 +82,31 @@ public function step_3(): void
             ]
         );
     }
+
+    public function step_4(): void
+    {
+        // check for missing mandatory "title" MD for series
```

Comment:
This step would not be necessary, but as for a double-checker I would say, that is ok.
A small typo:
```suggestion
        // check for missing mandatory "title" MD for event
```

Reply:
@chfsx Ping! 
The small suggestion could be applied, and then we could get it merged!